solvers/DPStokes/mobility.h

```diff
@@ -44,30 +44,110 @@ class DPStokes: public libmobility::Mobility{
     dpstokes = std::make_shared<uammd_dpstokes::DPStokesGlue>();
   }
 
+  double configure_grid_and_kernels_xy(Parameters ipar){
+    double hydroRadius = ipar.hydrodynamicRadius[0];
+
+    // STEP 1: set w & beta based on kernel type (Table 1/2)
+    double w = 6.0; // note: should be an int but I'm scared of the consequences
+    double beta = 1.714;
+
+    // STEP 2: use c(beta*w) polyfit to get c
+      // note: polyfit is to c(beta*w), not just beta
+    double c_beta = polyEval(cbetam, beta*w);
+    double h = hydroRadius / (w*c_beta); // compute h using h = Rh/(w*c) and nx = Lx/h
+    double nx_unsafe = dppar.Lx/h; // not necessarily fft friendly
+
+    // STEP 3: find candidates for fft friendly nx
+    std::vector<int> nx_safe = fft_friendly_sizes(floor(nx_unsafe), 100, 10);
+    int m = nx_safe.size();
+
+    std::vector<double> errors(m);
+    std::vector<double> h_candidates(m);
+    std::vector<double> beta_candidates(m);
+
+    int n = 1000;
+    double start = w; // interpolation range
+    double end = 3*w;
+    std::vector<double> err_axis(n); // creates a linspace vector
+    h = (end-start)/(n-1);
+    std::cout << h << std::endl;
```

Comment:
Do not print random info. If you want to leave debug prints use UAMMD's log and give it a DEBUG level.